src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperResolver.cs

```diff
@@ -54,6 +54,6 @@ protected virtual async Task<TagHelperResolutionResult> GetTagHelpersAsync(Proje
             return new TagHelperResolutionResult(results, Array.Empty<RazorDiagnostic>());
         }
 
-        protected virtual Task<TagHelperResolutionResult> GetTagHelpersAsync(Project workspaceProject, RazorProjectEngine engine) => GetTagHelpersAsync(workspaceProject, engine);
+        protected virtual Task<TagHelperResolutionResult> GetTagHelpersAsync(Project workspaceProject, RazorProjectEngine engine) => GetTagHelpersAsync(workspaceProject, engine, CancellationToken.None);
```

Comment:
Good find 👏 . This probably didn't cause problems because not all implementations use this.

Reply:
Actually my exp instance just freezes without this change.

Reply:
Nice! Wonder why we didn't get a warning logged on us for this.